src/object/obj_internal.h

```diff
@@ -44,6 +44,8 @@
  */
 #define IO_BYPASS_ENV	"DAOS_IO_BYPASS"
 
+#define HIGH_BIT (1UL << 63)
```

Comment:
seems better to use more specific name  (HIGH_BIT seems a little too generic?), and some comment about the usage?